src/ExternalApi/Isite/Domain/BaseIsiteObject.php

```diff
@@ -49,7 +49,7 @@ public function __construct(
         string $projectSpace,
         string $parentPid,
         string $brandingId,
-        string $image,
+        IsiteImage $image,
```

Comment:
If we want to be able to render pages with invalid images I think this should allow null
```suggestion
        ?IsiteImage $image,
```